rts/engine/game.cc

```diff
@@ -383,6 +383,9 @@ PlayerId RTSGame::MainLoop(const std::atomic_bool *done) {
           for (const auto &bot : _bots) {
               bot->Act(_env, true);
           }
+          if (_spectator != nullptr) {
+            _spectator->Act(_env);
+          }
```

Comment:
@yuandong-tian could you comment on whether this line is needed? The rest looks good to me.